css-values/Overview.bs

```diff
@@ -1609,11 +1937,12 @@ Type Checking</h4>
 
 		<li>
 			At ''*'',
-			check that at least one side is <<number>>.
```

Comment:
`<<integer>>` is a subset of `<<number>>`, but mentioning it explicitly should make the next line less surprising.